recipes/seqfu/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
you don't need to bump for this change, I can merge it without if you are ok with that.

Reply:
Ah perfect! Thanks

Reply:
@telatin I added spdx identifiers and the license file. In this process, I found that the license was annotated wrong. Can you double check this please?

Reply:
Thanks @bgruening, you're right, it's GPL!